tendrl-alerting.spec

```diff
@@ -0,0 +1,68 @@
+Name: tendrl-alerting
+Version: 1.2
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Tendrl Alerting
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/alerting
+
+BuildRequires: python-gevent
+BuildRequires: pytest
+BuildRequires: systemd
+BuildRequires: python-mock
+
+Requires: tendrl-commons
+
+%description
+Python module for Tendrl alerting
+
+%prep
+%setup
+
+# Remove bundled egg-info
+rm -rf %{name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/alerting
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/alerting
+install -m  0755  --directory $RPM_BUILD_ROOT%{_datadir}/tendrl/alerting
+install -Dm 0644 tendrl-alerting.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-alerting.service
+install -Dm 0644 etc/tendrl/alerting.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/alerting/alerting.conf.yaml
+install -Dm 0644 etc/tendrl/email.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/alerting/email.conf.yaml
+install -Dm 644 etc/*.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/alerting/.
```

Comment:
Please re-check https://github.com/Tendrl/node-agent/blob/develop/tendrl-node-agent.spec#L49 and add the logging related files